app/index.js

```diff
@@ -6,9 +6,31 @@ const yeoman = require('yeoman-generator');
 const _s = require('underscore.string');
 
 module.exports = yeoman.Base.extend({
+	constructor: function () { // eslint-disable-line babel/object-shorthand
```

Comment:
No idea what's causing that, but I just switched it to ES2015 classes: https://github.com/sindresorhus/generator-nm/commit/964ba1f696bd1c90b34cd786d18a93cf5a99a26d